src/libbpf.c

```diff
@@ -9196,6 +9196,37 @@ const char * libbpf_with_debugfs_location_prefix(const char * file)
 	return path;
 }
 
+// Customization:
+static int probe_exsits(const char * name, bool uprobe, bool retprobe)
```

Comment:
1. typo: exsits -> exists
2. please name it with `debugfs_` prefix

Reply:
Done